mentorship ios/Service/Networking/ProfileAPI.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ProfileAPI.swift
+//  Created on 22/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Foundation
+import Combine
+
+class ProfileAPI: ProfileService {
+    private var cancellable: AnyCancellable?
+    let currentProfile = ProfileViewModel().getProfile()
+    
+    func getProfile(completion: @escaping (ProfileModel.ProfileData) -> Void) {
+        //get auth token
+        guard let token = try? KeychainManager.getToken() else {
+            return
+        }
+        print(token)
+        
+        //parallel request for profile and home
+        cancellable = NetworkManager.callAPI(urlString: URLStringConstants.Users.user, token: token)
+            .receive(on: RunLoop.main)
+            .catch { _ in Just(self.currentProfile) }
```

Comment:
can we replace with `Just(ProfileViewModel().getProfile())` and remove local variable?

Reply:
Done

Reply:
@annabauza In cases like these, I shouldn't be making a separate network model right?